tests/pipeline/test_pipeline.py

```diff
@@ -35,12 +41,11 @@ def test_load_pipeline(self, tmpdir, snapshot):
 
         assert result.exit_code == 0
 
-        enriched_pipeline = load_yaml_file(Path(output_file_path))
+        enriched_pipeline = load_yaml(result.stdout)
```

Comment:
Wouldn't a `yaml.safe_load(resutl.stdout)` be sufficient here?

Reply:
yep, good point!